ProvisionSample/Program.cs

```diff
@@ -319,7 +319,6 @@ static void Main(string[] args)
                                 Console.WriteLine("\t{0}: {1}", report.Name, report.WebUrl);
                             }
                         }
-                        break;
```

Comment:
I believe this `break;` was appropriate and that we should keep it. It appears that the Run block was written such that an exit flag is initialized to false, and it's checked after the try/catch to call Run again, allowing the user to perform another operation. 

I don't believe this will compile with control falling through from one case (0) to another (default). If it did, we'd still end up flipping the exit flag to true, resulting in an immediate exit instead of a repeated prompt to perform another operation.

Am I missing anything here?